Makefile

```diff
@@ -182,31 +181,30 @@ $(BINDIR)/validate-trust-package: cmd/validate-trust-package/main.go pkg/fspkg/p
 	CGO_ENABLED=0 go build -o $@ $<
 
 .PHONY: depend
-depend: $(BINDIR)/deepcopy-gen $(BINDIR)/controller-gen $(BINDIR)/ginkgo $(BINDIR)/kubectl $(BINDIR)/kind $(BINDIR)/helm $(BINDIR)/kubebuilder/bin/kube-apiserver
+depend: $(BINDIR)/deepcopy-gen
+depend: $(BINDIR)/controller-gen
+depend: $(BINDIR)/boilersuite
+depend: $(BINDIR)/kind
+depend: $(BINDIR)/helm-docs
+depend: $(BINDIR)/helm
+depend: $(BINDIR)/ginkgo
+depend: $(BINDIR)/kubectl
+depend: $(BINDIR)/kubebuilder/bin/kube-apiserver
 
 $(BINDIR)/deepcopy-gen: | $(BINDIR)
-	go build -o $@ k8s.io/code-generator/cmd/deepcopy-gen
+	cd hack/tools && go build -o $@ k8s.io/code-generator/cmd/deepcopy-gen
 
 $(BINDIR)/controller-gen: | $(BINDIR)
-	go build -o $@ sigs.k8s.io/controller-tools/cmd/controller-gen
+	cd hack/tools && go build -o $@ sigs.k8s.io/controller-tools/cmd/controller-gen
```

Comment:
I think it makes sense to control the version of `controller-gen`, as the version is included in the CRDs.

Reply:
It is still versioned, in the hack/tools/go.mod file.

Reply:
Thanks for the explanation! Still learning the setup.